content/company-info-and-process/communication/code_of_conduct.md

```diff
@@ -284,9 +284,31 @@ We regularly implement and update procedures to prevent us from violating intern
 
 ## Raising concerns
 
-If you have a question or concern about a violation of the Code, speak up. Contact your manager, the People Team at people-ops@sourcegraph.com or jonathan.ayers@sourcegraph.com, or the Legal Team at legal@sourcegraph.com or tammy@sourcegraph.com. You can also use Slack, set up a meeting, or call them directly.
+All of us here at Sourcegraph are expected to perform our jobs ethically. If you have any questions or concerns about a violation of our Code of Conduct, we ask that you please come forward.
 
-If you want to remain anonymous, you can report your observations, concerns, and any suspected violations through our 24/7 third-party hotline at (toll free, 24/7 accessible phone number pending) or (website pending). The third-party hotline will share your report with limited members of the People and Legal Teams so we can look into and respond to your concerns.
```

Comment:
@CecilyBlack I assume that this sentence will still be true with Convercent:  "The third-party hotline will share your report with limited members of the People and Legal Teams so we can look into and respond to your concerns." If so, let's include it so that we set the right expectations. It's not like their report will go into the ether. It will be shared with limited members of the People and Legal teams (just without their name, i.e., anonymously), right?